src/App.tsx

```diff
@@ -1,31 +1,112 @@
+import { AnimatePresence } from 'framer-motion';
 import { useState } from 'react';
-import Counter from './components/Counter/Counter';
+
 import SearchForm from './components/SearchForm/SearchForm';
 import GenreSelect from './components/GenreSelect/GenreSelect';
+import MovieTile from './components/MovieTile/MovieTile';
+import MovieDetails from './components/MovieDetails/MovieDetails';
+import SortControl from './components/SortControl/SortControl';
```

Comment:
you can use named exports and re-export for components
create index file in components folder
`export * from './SearchForm'`